src/data/navigation/navigation.json

```
@@ -297,7 +272,7 @@
     }
   },
   "utilities": {
```

```suggestion
  "patterns": {
```

Sorry I missed this the first time, we need to upate this also to get the links to update correctly.

---

Fixed

---

updated